src/components/queue/QueueList/QueueList.tsx

```diff
@@ -73,10 +79,23 @@ const QueueList: FC<QueueListProps> = ({queue, showCompletedTickets}) => {
                     <CircularProgress/>
                 </Stack>}
                 <Stack spacing={1}>
-                    {sortedTickets && sortedTickets.map((ticket, index) => <QueueListItem key={ticket!.id}
+                    {beforeCutoff && beforeCutoff.map((ticket, index) => <QueueListItem key={ticket!.id}
                                                                                           queue={queue}
                                                                                           ticket={ticket!}
                                                                                           position={index + 1}/>)}
+                    {queue.isCutOff &&  <>
```

Comment:
hmm, this only renders the cut-off tickets if the queue is cut off; is it possible to un-cut off the queue and lose these tickets?

Reply:
it seems that the backend validates this; the only case i'm worried about is the one detailed in the sole comment on [the other pr](https://github.com/nthnluu/hours-backend/pull/12#pullrequestreview-924925676)